inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -369,7 +338,7 @@
                         "type": "string"
                     },
                     "cnum": {
-                        "type": "array"
+                        "type": "string"
```

Comment:
@tomaszgy, @bittirousku: this change generates this error: https://travis-ci.org/inspirehep/inspire-next/jobs/132541720#L2200.

Shall we revert this, or amend the CNUM rule in DoJSON?